src/magnify/apps/core/serializers/users.py

```diff
@@ -0,0 +1,42 @@
+"""Serializers for users views"""
+
+from rest_framework import serializers
+
+
+class UserCreateSerializer(serializers.Serializer):
```

Comment:
Why not using a ModelSerializer mapping it with the user model ? Doing this you have only one serializer https://www.django-rest-framework.org/api-guide/serializers/#modelserializer

Reply:
Tks a lot for this review, I'm working on it
I still have some issues with the response from the API. In order to have good documentation, I'm creating a serializer for each possible response, which is very heavy. Do you know if there is a better way to handle this or should I just not care ?